install.go

```diff
@@ -4,20 +4,40 @@ import (
 	"fmt"
 	"os"
 	"os/exec"
+
+	"health_monitor/utils"
+)
+
+const (
+	tempPath = "/tmp/monitor"
 )
 
 func install() {
 	if os.Getegid() != 0 {
-		fmt.Println("Please run the program with sudo privileges")
+		utils.Perror("Please run the program with sudo privileges")
 		os.Exit(1)
 	}
-	fmt.Println("Installing ssdeep package...")
-	command := exec.Command("/bin/sh", "-c", "apt-get install ssdeep")
+
+	complete := true
+	fmt.Println("[-]Installing ssdeep package...")
+	command := exec.Command("/bin/sh", "-c", `wget http://downloads.sourceforge.net/project/ssdeep/ssdeep-2.13/ssdeep-2.13.tar.gz; \
+		tar zxvf ssdeep-2.13.tar.gz; cd ssdeep-2.13; ./configure; make; make install `)
+	command.Stdout = os.Stdout
+	command.Dir = tempPath
 	if err := command.Run(); err != nil {
-		fmt.Println(err)
+		utils.Perror("Unable to install ssdeep")
+		complete = false
```

Comment:
Add a `Perror` command here to print `wget http://downloads.sourceforge.net/project/ssdeep/ssdeep-2.13/ssdeep-2.13.tar.gz; tar zxvf ssdeep-2.13.tar.gz; cd ssdeep-2.13; ./configure; make; make install` for user to run it manually if it does not run from the app itself.